samples/net/wifi/apsta_mode/boards/esp32_devkitc_wroom_procpu.overlay

```diff
@@ -1,10 +1,3 @@
 &wifi {
 	status = "okay";
 };
-
-/ {
```

Comment:
Please, restore this otherwise it won't even build for `esp32_devkitc_wroom` board.